### DISCUSSION TITLE
Model Results failed to initialize

### DISCUSSION THREAD

**Discussion by conghung** [DHI/modelskill#130] [Q&A] [answered]
When I run to initialize the ModelResult with param is file path then throwed the error like below:
![FmSkill_Issue_ModelResult](https://user-images.githubusercontent.com/10082268/173111617-afbeb0ae-200f-40fc-ac86-bd8134c5860f.png)
I just use the unit test file provided from source code.

**Comment by jsmariegaard** [comment, marked as answer]
Hi @conghung, sorry - I somehow missed your post earlier. This problem is due to the update of MIKE IO to 1.0. If you update FMskill again it should also update MIKE IO and the problem should be solved.
